app/src/main/java/org/simple/clinic/login/applock/ConfirmResetPinDialog.kt

```diff
@@ -30,6 +44,13 @@ class ConfirmResetPinDialog : AppCompatDialogFragment() {
           val view = dialog.ownerActivity.findViewById<View>(android.R.id.content)
           val snackbar = Snackbar.make(view, getString(R.string.applock_reset_pin_waiting), Snackbar.LENGTH_INDEFINITE)
           snackbar.show()
+
+          userSession.startForgotPinFlow(1)
+              .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Can you statically import `mainThread()`?

Reply:
Oh I see you've replaced `1` with a property. Why not just use a named param?

Reply:
That actually would be better. Will fix.